lib/pharos/configuration/host.rb

```diff
@@ -82,6 +82,9 @@ def labels
         labels = @attributes[:labels] || {}
 
         labels['node-address.kontena.io/external-ip'] = address
+        pa = private_address || private_interface_address
```

Comment:
IMHO `pa` ->  `internal_ip` would make this more readable.

Reply:
done